docs/config.toml

```diff
@@ -0,0 +1,166 @@
+baseURL = "https://kubernetes-sigs.github.io/bom"
```

Comment:
As discussed, we will stage the first deployment using the github.io domain before sorting out DNS resolution for `bom.sigs.k8s.io`